pkg/network/netlink/testutil/conntrack.go

```diff
@@ -57,10 +62,13 @@ func getDefaultInterfaceName(t *testing.T) string {
 // SetupDNAT6 sets up a NAT translation from fd00::2 to fd00::1
 func SetupDNAT6(t *testing.T) {
 	ifName := getDefaultInterfaceName(t)
+	cd, err := curDir()
+	require.NoError(t, err)
 	cmds := []string{
 		"ip link add dummy1 type dummy",
 		"ip address add fd00::1 dev dummy1",
 		"ip link set dummy1 up",
+		fmt.Sprintf("%s/../testdata/wait_if.sh dummy1", cd),
```

Comment:
Would putting `wait_if.sh` in a `testdata` directly beneath `pkg/network/netlink/testutil` allows to remove the `curDir()` hacks?

Reply:
That doesn't work. The relative path may be off the current working dir, which can change from env to env. The curDir hack seems to work everywhere.

Reply:
I copied it from here btw: https://github.com/DataDog/datadog-agent/blob/main/pkg/network/http/testutil/testutil.go#L77

Reply:
We use relative paths in other tests. Are they broken? https://github.com/DataDog/datadog-agent/blob/b5bed4dc6a5b08a50346e614b386ea67e37695de/pkg/network/port_test.go#L102